docs/getting_started_jax.ipynb

```diff
@@ -0,0 +1,638 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d0ABTeSA_QZh"
+      },
+      "source": [
+        "# Shardy Guide for JAX Users\n",
+        "\n",
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/shardy/blob/main/docs/getting_started_jax.ipynb)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Shardy is a new propagation system being introduced into the XLA stack, and below we want to introduce any JAX users to:\n",
+        "\n",
+        "1. What has changed in JAX\n",
+        "2. Why Shardy?\n",
+        "3. Future plans\n",
+        "\n",
+        "This is meant for JAX users who use `jax.jit` for running training/inference models across more than 1 GPU or TPU (batch parallelism, megatron, ZeRO, etc). They would be using things like `PartitionSpec`s and `NamedSharding`s."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 1. What has changed in JAX?"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### State of JAX before: GSPMD"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Prior to Shardy, JAX users who partitioned their models across models across multiple devices used [GSPMD](http://go/arxiv/2105.04663) behind the scenes.\n",
+        "\n",
+        "GSPMD is the propagation+partitioning system that lives in the middle of the XLA pipeline. It operates on HLO - the IR that comes after StableHLO (the program you get after running `jax.jit.lower`).\n",
+        "\n",
+        "JAX doesn't run GSPMD directly, but encodes instructions into the StableHLO IR for GSPMD to read later on.\n",
+        "\n",
+        "But before we go any further, let's introduce our working example.\n",
+        "\n",
+        "Note that this example should be executed in an environment with 8 devices. Make sure you have installed `jax>=0.4.35`."
```

Comment:
Let's remove `Note that this example should be executed in an environment with 8 devices.` since they can run on one CPU and fake the extra devices.